### PR TITLE
Feature: Landing Page

### PR DESCRIPTION
**Pull request by BodaNabeel**
I have added landing page for Indic-Subtitler, also I have shifted upload component which was present at "/" (default route) to "/home" and set "/" for landing page

### CODE REVIEW

File: ui/tailwind.config.js [hunk 1]
```diff
@@ -12,6 +12,14 @@ module.exports = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      colors: {
```
[the remark this answers is not in the document]
Comment (kurianbenoy) [reply ui/tailwind.config.js:15]: @BodaNabeel can you check whether you can do anything about this feedback?

**Review by kurianbenoy — APPROVED** [review]
I loved the UI. Good job Nabeel.

File: ui/tailwind.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (BodaNabeel) [reply ui/tailwind.config.js:15]: Looked into this, mistakenly I'd added same sort of code twice. Let me fix it and commit the changes.

File: ui/src/components/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "Malayalam",
+    "Marathi",
+    "Odia",
+    "Punjabi",
+    "Tamil",
+    "Telgu",
+    "Urdu",
+  ];
+  return (
+    <div className="carousel carousel-center max-w-full p-4 space-x-4 bg-gray-100 rounded-box mt-12 border-4 border-primary">
```
Comment (aldrinjenson) [inline ui/src/components/Carousel.jsx:20]: Small thought: Can we use some npm library to make this auto-scroll after a few seconds?

Comment (BodaNabeel) [reply ui/src/components/Carousel.jsx:20]: I was thinking the same, I think the carousel can be revamped into better one so, I think we should take some time to rethink this section altogether.

**Comment by BodaNabeel** [comment]
I will increase the vertical spacing between sections and commit it. 
Further I have query regarding the horizontal padding for mobile devices,  are you referring horizontal padding only for footer or every section?

File: ui/src/components/Carousel.jsx (hunk 2 above)
Comment (kurianbenoy) [reply ui/src/components/Carousel.jsx:20]: image

Thanks for sharing this idea @BodaNabeel

Comment (BodaNabeel) [reply ui/src/components/Carousel.jsx:20]: Alright.

**Review by aldrinjenson — APPROVED** [review]

**Comment by aldrinjenson** [comment]
Hey @BodaNabeel just a minor suggestion:

Can we please make the "Try Now" buttons point (as a link) to the dashboard route ?
and for now, let's just change "Sign Up" to "Dashboard" or even "Generate" maybe ? With all buttons acting as link to the dashboard route. 

image



@kurianbenoy what do you think? Authentication could be too much at this stage right? Maybe we can think about it at the very end as a low priority one?

**Review by kurianbenoy — APPROVED** [review]
@BodaNabeel can you point the button to the file upload screen as Aldrin pointed out?

**Comment by kurianbenoy** [comment]
Merging the PR

**Pull request merged**